config/config.go

```diff
@@ -143,14 +143,14 @@ func (c *Config) IsAzure() bool {
 	return isAzureHost || c.AzureResourceID != ""
 }
 
-// IsGcp returns true if client is configured for GCP
+// IsGcp returns if the client is configured for Databricks on Google Cloud.
 func (c *Config) IsGcp() bool {
 	return strings.Contains(c.Host, ".gcp.databricks.com")
 }
 
-// IsAws returns true if client is configured for AWS
+// IsAws returns if the client is configured for Databricks on AWS.
 func (c *Config) IsAws() bool {
-	return !c.IsAzure() && !c.IsGcp()
+	return strings.Contains(c.Host, ".databricks.com") && !c.IsAzure() && !c.IsGcp()
```

Comment:
we have other top-level domains for AWS (or PVC), this won't be sufficient and may break some deployments

Reply:
Good point. Modified to check for host not empty in this function.

If the host is empty, `IsAws` should also return false because we don't know (different from current behavior).